README.md

```diff
@@ -1,17 +1,62 @@
 # Getting Started with the Angular Stock Chart Component
 
-A quick-start project to create a new Angular application using the Angular CLI toolchain and add the Syncfusion Stock Chart component to it. This project contains simple code customization, as well as some important features like enabling tooltips and crosshairs.
-
-Refer to the following documentation on the Syncfusion Angular Stock Chart component: 
-https://ej2.syncfusion.com/angular/documentation/stock-chart/getting-started    
-
-Check out this online example of the Angular Stock Chart component: 
-https://ej2.syncfusion.com/angular/demos/#/bootstrap5/stock-chart/default
+The [Angular Stock Chart](https://www.syncfusion.com/angular-components/angular-stock-chart?utm_source=github&utm_medium=listing&utm_campaign=angular-stock-chart-github-samples) is a well-crafted, easy-to-use financial charting package to track and visualize stock price of any company over a specific period using charting(such as Candlestick, OHLC, HiLo, etc.) and range tools. A quick-start project to create a new Angular application using the Angular CLI toolchain and add the Syncfusion Stock Chart component to it. This project contains simple code customization, as well as some important features like enabling tooltips and crosshairs.  
 
 ## Project prerequisites
 
 Make sure that you have the latest versions of NodeJS and Visual Studio Code in your machine before starting to work on this project.
 
 ### How to run this application?
 
-To run this application, you need to clone the `getting-started-with-the-angular-stock-chart-component` repository and then open it in Visual Studio Code. Now, simply install all the necessary react packages into your current project using the `npm install` command and run your project using the `ng serve` command.
+To run this application, you need to clone the `getting-started-with-the-angular--stock-chart--component` repository and then open it in Visual Studio Code. Now, simply install all the necessary react packages into your current project using the `npm install` command and run your project using the `ng serve` command.
+
+## Features and Benefits
+
+### Interactive Chart
```

Comment:
Can we add an interlink to these features?

Reply:
yes, have added